README.md

```diff
@@ -1,32 +1,86 @@
 # Docker MinIO
 
-This is a simple docker image for running a minio server. It is based on alpine linux and uses the minio and minio-client packages from the alpine package repository.
+This is a simple docker image for running a MinIO server. It is based on alpine linux and uses the `minio` and `minio-client` packages from the alpine package repository.
 
-The `/scripts/entrypoint.sh` script is used to start the minio server. It is possible to configure the container to create and populate the new bucket at startup by setting the `BUCKET_NAME` environment variable and adding the seed files to the folder defined by the `INITFILES_FOLDER` environment variable.
+The `/scripts/entrypoint.sh` script is used to start the MinIO server. It is possible to configure the container to create and populate the new bucket at startup by setting the `BUCKET_NAME` environment variable.
+
+## Bucket initialization
+
+You can use two different methods to initialize the bucket: the `FileSystem` initialization and the `Seed` initialization.
+
+## FileSystem initialization
+
+If the `INITFILESYSTEM_FOLDER` path is not empty, the files in the folder are copied to the `BUCKET_ROOT` destination folder as a startup filesystem for the MinIO server. **The folder must contain a valid MinIO filesystem structure.** **ATTENTION**: when the `INITFILESYSTEM_FOLDER` is used, the startup process takes care of checking that the `BUCKET_NAME` is configured accordingly with the final MinIO server configuration, performing the filesystem check (the `BUCKET_NAME` folder is present in the `BUCKET_ROOT` folder) and the MinIO client check (the `mc ls ${MC_ALIAS}/${BUCKET_NAME}` command does not return an error).
+
+If this initialization method is used, **no other initialization actions are performed**. The final bucket content is only the data present in the `INITFILESYSTEM_FOLDER` folder.
```

Comment:
I am wondering if we should support both methods, seed files +  bucket files, they shouldn't collide and can be processed in a chain.

Reply:
Implemented. [This](https://github.com/sparkfabrik/Docker-MinIO/pull/14/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R7) is the new _seed_ logic.

If the bucket exists at the startup, no seed process will be executed. In other words, if  the `/data` folder is a valid MinIO filesystem with an already created bucket, nothing happens. To test this scenario, you can mount a local data folder in `/data` container mount point and exec `make start` twice. At the first run the container will seed the mounted `/data` folder, at the second run the container will skip the initialization process.